.vscode/launch.json

```diff
@@ -11,7 +11,7 @@
             "preLaunchTask": "build",
             // If you have changed target frameworks, make sure to update the program path.
             "program": "${workspaceFolder}/SQRLDotNetClientUI/bin/Debug/netcoreapp3.1/SQRLDotNetClientUI.dll",
-            "args": [],
+            "args": ["sqrl://www.grc.com/sqrl?nut=SZNWC8znDcvovhbaT8anQg"],
```

Comment:
Is this on purpose, Jose, or is it a leftover from testing?

Reply:
That was for testing on the Mac. I'll remove it.